### ISSUE TITLE
When used with Accessibility Developer Tools, the text / background contrast is hard to read.

### ISSUE THREAD

**Issue by alan-cole-salsa-digital** [mauricecruz/chrome-devtools-zerodarkmatrix-theme#137]
When using the Google Chrome plugin Accessibility Developer Tools ( https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb ) the Accessibility Properties tab text is black, on a dark background. The text should be light as per other tabs in that section.
![screen shot 2015-08-12 at 12 02 20 pm](https://cloud.githubusercontent.com/assets/12739575/9215182/504f1a16-40ea-11e5-9829-bc66ba8c7bf1.png)

**Comment by mauricecruz** [comment]
I'm afraid I don't have any good permanent solutions for you. Devtools extensions styles are sandboxed from each other so there'd be nothing I can do change the text color from my theme.

The background is inherited from the default pane styles. If I modify it, all the other panes will change as well.

But there is a way to change the color for just this pane. It just involves a few manual steps if you're willing.
* Open a (devtools session of chrome devtools)[http://s10wen.com/blog/2012/11/04/styling-the-chrome-dev-tools/]. Let's call the one you'll be using the accessibility tools on `primary` and the devtools of the devtools one `secondary`
* On `primary`, navigate to the accessibility extension on the sidebar pane. This will inject the UI into devtools
* On `secondary` open the console panel and set the dropdown to `sidebar.html`
* On the console in `secondary`, type `$('html').setAttribute('style','background-color: white')`
* This will set that specific extensions background to white and make the text more visible.

**note:** This will have to be done each time you start a new devtools session.

![image](https://cloud.githubusercontent.com/assets/1640686/9282432/5a332f5c-429a-11e5-971e-417f95afea80.png)

![image](https://cloud.githubusercontent.com/assets/1640686/9282395/21a5db76-429a-11e5-8007-d97ea88cc886.png)

**Comment by alan-cole-salsa-digital** [comment]
Thanks for the suggestion!
